docs/arts-and-crafts.md

```diff
@@ -28,10 +28,23 @@ into real world art! Please drop us a message if you know another tool to be add
 ]
 ```
 
+## Toys and Pets
+
+```codecard
+[
+    {
+        "name": "KittyWumpus",
+        "description": "A mechcanical adventure companion that joins with you in your game.",
```

Comment:
Suggest an more interesting description if you like.

Reply:
Mech? :)